### PR TITLE
Ubuntu single upload visualization

### PR DESCRIPTION
**Pull request by Sammers21**
Preview: 
image

Related to #431.

### CODE REVIEW

**Comment by Sammers21** [comment]
@olegmoz, review the PR, please.

**Comment by Sammers21** [comment]
@g4s8, review the PR, please.

File: benchmarks/index.html [hunk 1]
```diff
@@ -1,21 +1,103 @@
...
+                    })
+            )
+        ).filter(el => el !== undefined)
+            .sort((one, another) => compareDottedString(one.version, another.version))
+
+
+        // Draw Chart
+        let labels = results.map(json => json.version)
+        let dt = results.map(json => json.results.docker["single-upload"].artipie.images[0].ubuntu)
+        let ctx = document.getElementById('docker_single_upload').getContext('2d');
```
Comment (g4s8) [inline benchmarks/index.html:62]: @Sammers21 I don't think we need to hard-code Docker benchmarks in code, how we will switch between benchmarks when we have many of them?

File: benchmarks/index.html [hunk 2]
```diff
@@ -1,21 +1,103 @@
...
+        new Chart(ctx, {
+            type: 'bar',
+            data: {
+                labels: labels,
+                datasets: [{
+                    label: 'Docker single upload time',
```
Comment (g4s8) [inline benchmarks/index.html:68]: @Sammers21 and same here: "Docker..." hardcoded

**Review by g4s8 — CHANGES_REQUESTED** [review]
@Sammers21 please check my comments

File: benchmarks/index.html (hunk 1 above)
Comment (Sammers21) [reply benchmarks/index.html:62]: @g4s8, it is the first step. Anything else will come later

File: benchmarks/index.html (hunk 2 above)
Comment (Sammers21) [reply benchmarks/index.html:68]: @g4s8, it is the first step. Anything else will come later

**Review by g4s8 — APPROVED** [review]
@Sammers21 ok, let's merge

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#550` is not assigned, can't get performer

**Pull request merged**